invisible_cities/io/mcinfo_io_test.py

```diff
@@ -341,137 +340,99 @@ def test_read_mcparticles_df(mc_particle_and_hits_nexus_data):
     assert_allclose(init_mom.tolist(), p)
 
 
-def test_load_sensors_data(mc_sensors_nexus_data):
-    efile, pmt0_first, pmt0_last, pmt0_tot_samples, sipm_id, sipm = mc_sensors_nexus_data
-
-    mcsensors_dict = load_mcsensor_response(efile)
-
-    waveforms = mcsensors_dict[0]
+@fixture(scope = 'module')
+def new_format_mcsensor_data(ICDATADIR):
+    sensor_binning = (0.003 * units.mus,
+                      0.010 * units.mus,
+                      0.001 * units.mus)
+    evt_no         = 1280
+    sns_list       = [20005  , 20016  , 57  , 51  ]
+    charge_list    = [    1  ,     1  ,  1  ,  1  ]
+    time_list      = [   48.0,    27.0, 90.0, 50.0]
+    efile = os.path.join(ICDATADIR, 'NextFlex_mc_sensors.h5')
+    return efile, sensor_binning, evt_no, sns_list, charge_list, time_list
 
-    sns_number = 0
-    wvf        = waveforms[sns_number]
-    bins       = [t / wvf.bin_width for t in wvf.times]
-    samples    = list(zip(bins, wvf.charges))
 
-    assert samples[0]   == pmt0_first
-    assert samples[-1]  == pmt0_last
-    assert len(samples) == pmt0_tot_samples
+@fixture(scope = 'module')
```

Comment:
Maybe you can manually construct the exact binning dataframe you expect for both new and old format (since it is 2-3 rows) so the check is more obvious?